Docs/API_design_doc.md

```diff
@@ -32,11 +32,12 @@
 
     * Response Data:
 
-        | Name | Type | Description |
-        |----- |------|-------------|
-        | status | int | Status code |
-        | error | JSON | Error information |
-        | data | JSON (list) | [Recommended data](#recommended_data) |
+      | Name | Type | Description                                              |
+      |----- |----------------------------------------------------------|-------------|
+      | status | int | Status code                                              |
+      | error | JSON | Error information                                        |
+      | api_version | string | The API version parsed from environment variables |
```

Comment:
> The API version parsed from environment variables

The source of the version can be obtained from the code logic. We should explain the meaning of the api version in the doc such as large versions for major upgrades, patch version for bugfix

Reply:
A short version number description has been added, detailed in HISTORY.rst